### PR TITLE
Improve documentation of AllowedGroups, AllowedUsers and IgnoredPaths

### PR DESCRIPTION
**Pull request by mhabrnal**
Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>

### CODE REVIEW

File: doc/abrt-CCpp.conf.txt [hunk 1]
```diff
@@ -43,17 +43,19 @@ SaveFullCore = 'yes' / 'no' ...::
 
 IgnoredPaths = /path/to/ignore/*, */another/ignored/path* ...::
    ABRT will ignore crashes in executables whose absolute path matches one of
-   specified patterns.
+   specified glob patterns listed in the comma separated IgnoredPaths list.
```
Comment (jfilak) [inline doc/abrt-CCpp.conf.txt]: any of the glob patterns listed in the comma separated list.

File: src/hooks/CCpp.conf [hunk 2]
```diff
@@ -54,13 +54,14 @@ SaveFullCore = yes
 # StandaloneHook = yes
 
 # ABRT will ignore crashes in executables whose absolute path matches one of
-# specified patterns.
+# specified glob patterns listed in the comma separated IgnoredPaths list.
```
Comment (jfilak) [inline src/hooks/CCpp.conf]: I think examples would make it clearer.

**Comment by mhabrnal** [comment]
Updated

**Comment by jfilak** [comment]
Thank you.

**Pull request merged**